multimodal/src/autogluon/multimodal/optimization/RKDLoss.py

```diff
@@ -0,0 +1,37 @@
+# Reference: https://github.com/lenscloth/RKD/blob/master/metric/loss.py
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from .utils import pdist
+
+class RKDDistance(nn.Module):
+    def forward(self, student, teacher):
```

Comment:
Add docstrings and type hints to explain what this function does and what arguments are required.

Reply:
Done.